src/NuGet.Core/Microsoft.Build.NuGetSdkResolver/NuGetSdkResolver.cs

```diff
@@ -73,14 +73,14 @@ public override SdkResult Resolve(SdkReference sdkReference, SdkResolverContext
             // Escape hatch to disable this resolver
             if (DisableNuGetSdkResolver.Value)
             {
-                return factory.IndicateFailure(errors: null, warnings: new List<string>() { Strings.Error_DisabledSdkResolver });
```

Comment:
@Forgind 
Are there any tests need to be updated as well?

Reply:
I wouldn't suspect so unless you've added them since my previous PR, but if CI fails, I can look into any tests that need updates.